requirements.txt

```
@@ -0,0 +1 @@
+pygame
```

I think we should specify versions
```suggestion
pygame~=2.5
```

---

Agreed, just commited the suggestion